.github/workflows/test_prs.yml

```diff
@@ -66,3 +66,11 @@ jobs:
       - name: Running tests
         run: npm test
         working-directory: server
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
+        with:
+          configuration-path: .github/pr-labeler.yml # optional, .github/pr-labeler.yml is the default value
```

Comment:
you can remove the comment if it's not necessary

Reply:
Ill remove next PR :)